static/app/views/settings/components/settingsNavigation.tsx

```diff
@@ -38,9 +38,7 @@ class SettingsNavigation extends React.Component<Props> {
 
   componentDidCatch(error: Error, errorInfo: React.ErrorInfo) {
     Sentry.withScope(scope => {
-      Object.keys(errorInfo).forEach(key => {
-        scope.setExtra(key, errorInfo[key]);
-      });
+      scope.setExtra('componentStack', errorInfo.componentStack);
```

Comment:
componentStack is the property that comes with React.ErrorInfo so no need to loop